sources/packages/web/src/services/http/dto/StudentAccountApplication.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { CreateStudentAPIInDTO } from "./Student.dto";
+
+export interface CreateStudentAccountApplicationAPIInDTO {
+  submittedData: unknown;
+}
+
+export interface StudentAccountApplicationSummaryAPIOutDTO {
+  id: number;
+  fullName: string;
+  submittedDate: Date;
+}
+
+export interface StudentAccountApplicationAPIOutDTO {
+  id: number;
+  submittedData: unknown;
+}
+
+export interface HasPendingStudentAccountApplicationAPIOutDTO {
```

Comment:
The name of this DTO sounds like a boolean variable. Any particular reason why the hasPendingApplication attribute cannot be moved to StudentAccountApplicationOutDTO above ?

Reply:
This is an endpoint dedicated to the student and this is all the information that a student will be able to get from his application.
I did not get the idea of "move", but if the suggestion is about renaming it then I believe that would be ok.
Please clarify.

Reply:
Yes the suggestion is about renaming.

Reply:
Cool, I will rename it 👍

Reply:
DTO renamed.